app/models/transfer_package.rb

```diff
@@ -0,0 +1,50 @@
+class TransferPackage < ActiveRecord::Base
+  self.primary_key = :id
+
+  belongs_to :receiver_institution,
+    class_name: "Institution",
+    primary_key: "uuid"
+
+  has_many :sample_transfers
+
+  after_initialize do
+    self.id ||= SecureRandom.uuid
```

Comment:
CDx uses `Guid.new` everywhere but that gem is no longer maintained, and it looks like it generates invalid UUIDs (e.g. all 16 bytes are fully random). Let's start moving to the proper UUIDs.